README.md

```diff
@@ -47,7 +49,8 @@ git push
 
 ```commandline
 cd ~/repos/hello-robot.github.io
-mike serve
```

Comment:
I think you need `mike serve` to be able to visualize older versions like 0.1, 0.2, right?